src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -272,6 +281,11 @@ on_exit () {
 trap on_exit EXIT
 
 echo ===========================================================================
+if [ "${warningHeader}" != '' ]
```

Comment:
What do you think about doing something like the following:
```
const echoWarning = `echo '${warningHeader}'`;
...
return `#!/bin/sh
...
echo ===========================================================================
${echoWarning}
echo 'Feature       : ${name}'
...
```

That way we don't need to add a condition in the script that checks an inlined string. In the case where there's a warning to print, we generate the script with the echo added. In the case where there's no warning to print, we omit the echo entirely and the script output is roughly the same as it was before except there's a blank line in the script itself (not in the script output when the script is executed).

Reply:
Sweet! Modified.